samples/slack-spring-boot-starter-sample/src/main/kotlin/com/kreait/slack/sample/SetPhotoCommandReceiver.kt

```diff
@@ -1,30 +0,0 @@
-package com.kreait.slack.sample
-
-import com.kreait.slack.api.SlackClient
-import com.kreait.slack.api.contract.jackson.SlackCommand
-import com.kreait.slack.api.contract.jackson.group.users.SetPhotoRequest
-import com.kreait.slack.broker.receiver.SlashCommandReceiver
-import com.kreait.slack.broker.store.Team
-import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.http.HttpHeaders
-import org.springframework.stereotype.Component
-import java.io.File
-
-@Component
```

Comment:
please don't delete this receiver, since setting the photo is not completely clear yet